src/gtest/test_merkletree.cpp

```diff
@@ -235,6 +238,29 @@ TEST(merkletree, vectors) {
     test_tree<ZCTestingIncrementalMerkleTree, ZCTestingIncrementalWitness>(root_tests, ser_tests, witness_ser_tests, path_tests);
 }
 
+TEST(merkletree, emptyroots) {
+    Array empty_roots = read_json(std::string(json_tests::merkle_roots_empty, json_tests::merkle_roots_empty + sizeof(json_tests::merkle_roots_empty)));
+    Array::iterator root_iterator = empty_roots.begin();
+
+    libzcash::EmptyMerkleRoots<64, libzcash::SHA256Compress> emptyroots;
+
+    for (size_t depth = 0; depth <= 64; depth++) {
+        expect_test_vector(root_iterator, emptyroots.empty_root(depth));
```

Comment:
The similarity in names between ``empty_roots`` and ``emptyroots`` tripped me up on the first pass here. It looks like ``empty_roots`` is a test vector and ``emptyroots`` is the API under test, so maybe renaming them to ``emptyroots_expected`` vs ``emptyroots`` would distinguish their intended use.